apps/nextjs-website/src/lib/fetch/types/PageSection.ts

```diff
@@ -300,7 +300,7 @@ const RowTextSectionCodec = t.strict({
   __component: t.literal('sections.row-text'),
   title: t.string,
   subtitle: t.union([t.string, t.null]),
-  body: t.union([t.string, t.null]),
+  body: t.string,
```

Comment:
```suggestion
  body: t.union([t.string, t.null]),
```
Any reason why this became required?

Make it optional again or, if it has to be required, make it so inside react-components as well.